### PR TITLE
calico_node/Makefile: Fix the names of images pushed by semaphore

### PR DESCRIPTION
**Pull request by tomdee**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -10,64 +9,54 @@ and v2.0). There only exists a single patch version (or pre-release
...
-   ```
-
+````
 4. Update the `RELEASE_STREAM` variable at the top of calico_node/Makefile to the new major/minor version.
-
-4. Test the changes locally then open a pull request, make sure it passes CI, then merge.
-
-5. run `make -C calico_node release` to create container images for calico/node. Push them by following the instructions that the make command prints.
-
+2. Test the changes locally then open a pull request, make sure it passes CI and get it reviewed.
```
Comment (caseydavenport) [inline RELEASING.md]: We restart at 2. again here?

Comment (tomdee) [inline RELEASING.md]: Yeah, because why not :smile: The numbers don't mean anything, just that they should be rendered as a numbered list.

Comment (caseydavenport) [inline RELEASING.md]: :'(

Comment (tomdee) [inline RELEASING.md]: I can change them all to 1. if you prefer?

**Pull request merged**